mks/model/stuf_02_04.py

```diff
@@ -373,7 +374,9 @@ def extract_identiteitsbewijzen(persoon_tree: Tag):
 
         if type_number == 10:
             # do not show nederlandse identiteitskaart older than 3 months. passpoort etc stays
-            if result_id['datumAfloop'] + relativedelta(months=+3) < datetime.now():
+            if not result_id['datumAfloop']:
+                logging.error(f"ID without a datumEindeGeldigheid. soort: {type_number}")
+            elif result_id['datumAfloop'] + relativedelta(months=+3) < datetime.now():
```

Comment:
waarom `months=+3` en niet gewoon `months=3`?

Reply:
Zag dit in de docs staan, en vond het eigenlijk wel slim om aan te geven dat je 3 maanden verder bedoeld.